templates/open-tracing.cfg.xml.erb

```diff
@@ -0,0 +1,22 @@
+<open-tracing xmlns="http://<%= scope.lookupvar('::repose::cfg_namespace_host') %>/repose/open-tracing-service/v1.0"
+ service-name="<%= @service_name %>">
+    <jaeger>
+    <%- if ! @udp_connection_port.nil? && ! @udp_connection_host.nil? -%>
```

Comment:
generally our recommendation is to treat ERB logic indentation as separate from the content. Indentation can be confusing with HTML/XML because they both use `<>`.  The caveat being inline variable population (lined 8-10 would stay as they are).  The rest should all stop being indented.

Almost every ERB block you have should be